src/viadot/sources/cloud_for_customers.py

```diff
@@ -284,7 +284,7 @@ def get_response(
         return handle_api_response(
             url=url,
             params=filter_params,
-            auth=(username, password),
+            auth=(username, password._secret_value),
```

Comment:
The way to do this in pydantic is to use `get_secret_value()` https://docs.pydantic.dev/latest/examples/secrets/